src/styles/styles.css

```diff
@@ -23,7 +23,7 @@
   [type="text"],
   [type="search"],
   [type="number"] {
-    @apply text-textDark p-2 bg-input leading-[1.4] w-full;
+    @apply text-text-primary p-2 bg-surface-ui border-surface-ui leading-[1.4] w-full;
```

Comment:
Request: Don't use surface colours for borders, we have border colours in the theme. You can always add a new one if needed.